aspnetcore/fundamentals/logging/loggermessage.md

```diff
@@ -136,12 +153,13 @@ Enable `IncludeScopes` in the console logger options:
 logging.AddConsole(options => options.IncludeScopes = true);
 ```
 
-Setting `IncludeScopes` is required in ASP.NET Core 2.0 apps to enable log scopes. Setting `IncludeScopes` via *appsettings* configuration files is a feature that will be available with the ASP.NET Core 2.1 release.
+Setting `IncludeScopes` is required in ASP.NET Core 2.0 apps to enable log scopes. Setting `IncludeScopes` via *appsettings* configuration files is a feature that is planned for the ASP.NET Core 2.1 release.
 
-The sample app also clears other providers and adds filters to reduce the logging output. This makes it easier to see the sample's log messages that demonstrate `LoggerMessage` features:
+The sample app clears other providers and adds filters to reduce the logging output. This makes it easier to see the sample's log messages that demonstrate `LoggerMessage` features:
 
 [!code-csharp[Main](loggermessage/sample/Program.cs?name=snippet1&highlight=22)]
 
+<!-- TODO Is this missing something? To create a field, the following code ???-->
 Create a field to hold a `Func` delegate for the log scope. The sample app creates a field called `_allQuotesDeletedScope` (*Internal/LoggerExtensions.cs*):
```

Comment:
It's not missing anything. It's meant to be an instruction. "[You] create a field to ..."

I'll try a re-word it to make it say something like ...

> To create the log scope, add a field to hold a `Func` delegate for the scope.